app/src/components/ContributionDetail.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <template v-if="contributions.length > 0">
+    <section v-for="contribution in contributions" :key="contribution.grantName" class="px-4 md:px-12">
+      <div class="py-4 md:py-6 lg:py-8 border-b border-grey-100">
+        <div class="grid grid-cols-12 gap-x-8 gap-y-4 items-center">
+          <!-- grand image + grant description + optional grant round -->
+          <article class="col-span-12 md:col-span-6 lg:col-span-5">
+            <!-- subgrid -->
+            <div class="grid grid-cols-4 items-center gap-x-4">
+              <!-- img -->
+              <div class="hidden lg:block col-span-1">
+                <!-- do not forget to link the figure to the grant ^^ -->
+                <figure class="aspect-w-16 aspect-h-9 shadow-light cursor-pointer">
+                  <img
+                    class="w-full h-full object-center object-cover group-hover:opacity-90"
+                    :src="contribution.grantLogoURI ? contribution.grantLogoURI : '/placeholder_grant.svg'"
+                  />
+                </figure>
+              </div>
+              <!-- text -->
+              <div class="col-span-4 lg:col-span-3">
+                <a href="#" class="link">{{ contribution.grantName }}</a>
+                <!-- if this tx was via a matching round show round -->
+                <br /><span class="mr-2">via</span>
+                <a href="#" class="link">{{ contribution.roundName }}</a>
+                <!-- end if -->
```

Comment:
what is this for ?

Reply:
@thelostone-mc I am not sure I follow? Are you asking why this value is here?

Reply:
I think he is referring to the fact that the comments indicate there should be an `if` condition, but code has no such conditional. So either the comments are outdated or the code is missing something

Reply:
Haha yeah that's pretty which what I had asked :P

Reply:
the idea was to NOT show this, when there is no round data. so there must be a condition